deps/amqp_client/priv/schema/amqp_client.schema

```diff
@@ -0,0 +1,129 @@
+%% ----------------------------------------------------------------------------
+%% RabbitMQ amqp_client TLS options
+%% ----------------------------------------------------------------------------
+
+{mapping, "amqp_client.ssl_options", "amqp_client.ssl_options", [
+    {datatype, {enum, [none]}}
+]}.
+
+{translation, "amqp_client.ssl_options",
+fun(Conf) ->
+    case cuttlefish:conf_get("amqp_client.ssl_options", Conf, undefined) of
+        none -> [];
+        _    -> cuttlefish:invalid("Invalid amqp_client.ssl_options")
+    end
+end}.
+
+{mapping, "amqp_client.ssl_options.verify", "amqp_client.ssl_options.verify", [
+    {datatype, {enum, [verify_peer, verify_none]}}]}.
+
+{mapping, "amqp_client.ssl_options.fail_if_no_peer_cert", "amqp_client.ssl_options.fail_if_no_peer_cert", [
```

Comment:
I might be mistaken but isn't `fail_if_no_peer_cert` a server only option, and new OTP 26 ssl will complain if set for a client?
https://github.com/rabbitmq/rabbitmq-website/pull/1933

Reply:
@gomoripeti you are correct, we should leave it out.

Reply:
Right I was a bit aggressive in my schema copy/pasting. I'll go through it and remove all server_options!